tests/test_validate_outputs.py

```diff
@@ -291,6 +291,12 @@ def test_healsparse_property_maps(self):
             self._num_bands
         )
 
+    def check_strip_footprints(self, catalog):
+        """Test that heavy footprints were stripped from the catalog."""
+        children = catalog[catalog['parent'] != 0]
```

Comment:
You may want to switch to double quotes (`"parent"`) for consistency, though it isn’t enforced here.